package.json

```diff
@@ -73,7 +73,7 @@
     "test:coverage": "nyc --cache-dir .nyc_output/.cache -r lcov -e .ts mocha 'test/unit/**/*.test.ts' && nyc report",
     "test:spec": "mocha 'test/spec/**/*.test.ts'",
     "test": "yarn run test:unit && yarn run test:spec",
-    "download-test-cases": "node --loader ts-node/esm test/downloadSpecTests.ts",
+    "download-spec-tests": "node --loader ts-node/esm test/downloadSpecTests.ts",
```

Comment:
@jeluard I noticed this got changed.  In the rest of our repos this is how the command is named so I switched it back but am not tied to the change.  Was a habit thing and after I pushed the commit I looked it up and noticed it was switched recently.  If you want to switch it back we can do that.  Just figured would be easier if it matches everywhere else.  What do you think?